doc/rtd/conf.py

```diff
@@ -67,11 +67,12 @@
 # of the sidebar.
 html_logo = 'static/logo.png'
 
-def generate_docstring_from_schema(app, what, name, obj, options, lines):
+def generate_docstring_from_schema(*args):
     """Override module docs from schema when present."""
-    if what == 'module' and hasattr(obj, "schema"):
+    (_, what, _, obj, _, lines) = args
+    if what == 'module' and hasattr(obj, "__doc__"):
         del lines[:]
-        lines.extend(get_schema_doc(obj.schema).split('\n'))
+        lines.extend(obj.__doc__)
```

Comment:
Minimally I think we need a split("\n") here because you are appending a multi-line string to lines (list). 

The current branch doens't show any module names in the bulleted list: `tox -e doc; xdg-open doc/rtd_html/topics/modules.html`

Reply:
This is fixed in commit  [e1b52e3b6196055](https://github.com/canonical/cloud-init/pull/1101/commits/e1b52e3b619605565311728108334e6da720c42a). I don't think further manipulation of the `__doc__` is required, so these functions can just be removed.